package.json

```diff
@@ -114,6 +118,11 @@
           "default": "# ${noteName}\n\n",
           "description": "Template for auto-created note files. Available tokens: ${noteName}, ${timestamp}. Timestamp is inserted in ISO format, i.e. 2020-07-09T05:29:00.541Z."
         },
+        "vscodeMarkdownNotes.newNoteFromSelectionReplacementTemplate": {
+          "type": "string",
+          "default": "[[${wikiLink}]]",
+          "description": "Template for generated links to notes created from selections. Available tokens: ${wikiLink}, ${noteName}."
```

Comment:
This is kind of an awkward description. Any suggestions?

Reply:
Hmm, OK, so this is what gets inserted into the active document after you effectively "cut" the selected text and "paste" it into the new note, right?

If that's the case, I might say:

```
"description": "After the selection is 'cut' from the active document, replacement text is 'inserted' into its place. Available tokens: ${wikiLink}, ${noteName}. The default insertion is a wiki-link to the new note containing the selected text."
```

Do you think it makes sense to have a token that represents the selected text, ie, if someone wants to copy / past (instead of cut / paste) to the new doc? Or does that workflow run counter to the intention of the PR?